sdk/iot/Azure.Iot.Hub.Service/tests/prerequisites/setup.ps1

```diff
@@ -117,4 +119,4 @@ if (-not (Test-Path $userTestAssetSettingsFileName))
     New-Item -ItemType File -Path $userTestAssetSettingsFileName -Value "{}" | Out-Null
```

Comment:
We don't need this file.

Reply:
I'll let @azabbasi take care of resource generation flow.